Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/HttpTriggerCSharp.cs

```diff
@@ -1,20 +1,31 @@
 using System.Net;
 using Microsoft.Azure.Functions.Worker;
-using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Extensions.Logging;
+#if NetCore
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+#elif NetFramework
+using Microsoft.Azure.Functions.Worker.Http;
+#endif
 
 namespace Company.Function
 {
     public class HttpTriggerCSharp
     {
-        private readonly ILogger _logger;
+        private readonly ILogger<T> _logger;
 
-        public HttpTriggerCSharp(ILoggerFactory loggerFactory)
+        public HttpTriggerCSharp(ILogger<T> logger)
         {
-            _logger = loggerFactory.CreateLogger<HttpTriggerCSharp>();
+            _logger = logger;
         }
 
         [Function("HttpTriggerCSharp")]
+#if NetCore
+        public IActionResult Run([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequest req)
```

Comment:
Authorization levels should be the same in both cases. AuthLevelValue is correct here I think because it will be set as a parameter to the template engine.

Reply:
Method choice should also be the same, I think.